daml-lf/spec/daml-lf-1.rst

```diff
@@ -3846,8 +3889,8 @@ BigNumeric functions
     unless both neighbors are equidistant, in which case round towards
     the even neighbor.
 
-  - ``'ROUNDING_UNNECESSARY'`` : Throw `ArithmeticError` if the exact result cannot be
-    represented.
+  - ``'ROUNDING_UNNECESSARY'`` : Throws ``ArithmeticError`` if the
```

Comment:
that change seems off, `ROUNDING_UNNECSSARY` does not throw. It causes the bulitin you pass it in to throw.

Reply:
I am not sure what is the issue here. 
I just added
- a "s" at the end of "Throw" 
- two "`" 
- and a newline

Reply:
With the `s` it sounds  to me like `ROUNDING_UNNECESSARY` is a primitive that throws rather than an argument which causes something else to throw. But that’s mostly gut feel so maybe my language intuition is just off here.

Reply:
OK. I revert back and remove the 's'. I also removing from some of the "Round" there were in the enumeration.